packages/frontend/graphql/src/schema.ts

```diff
@@ -30,12 +30,39 @@ export interface Scalars {
   DateTime: { input: string; output: string };
   /** The `JSON` scalar type represents JSON values as specified by [ECMA-404](http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf). */
   JSON: { input: Record<string, string>; output: Record<string, string> };
+  /** The `JSONObject` scalar type represents JSON objects as specified by [ECMA-404](http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf). */
+  JSONObject: { input: any; output: any };
```

Comment:
this seems to be irrelevant. Is it related to the json mode output?

Reply:
used in updateRuntimeConfigs